src/openrct2/ride/Vehicle.cpp

```diff
@@ -4238,27 +4238,27 @@ void Vehicle::UpdateTravellingBoat()
     UpdateMotionBoatHire();
 }
 
-static void loc_6DA9F9(Vehicle* vehicle, int32_t x, int32_t y, int32_t trackX, int32_t trackY)
+void Vehicle::Loc6DA9F9(const CoordsXY& coords, const CoordsXY trackCoords)
```

Comment:
```suggestion
void Vehicle::Loc6DA9F9(const CoordsXY& coords, const CoordsXY& trackCoords)
```

Reply:
I think we can name this function TryReconnectBoatToTrack